jlama-core/src/main/java/com/github/tjake/jlama/util/Downloader.java

```diff
@@ -23,6 +23,8 @@
 
 public class Downloader {
 
+    private static final String AUTH_TOKEN_ENV_VAR = "JLAMA_AUTH_TOKEN";
```

Comment:
According to HF docs this should be HF_TOKEN:

https://huggingface.co/docs/huggingface_hub/en/package_reference/environment_variables#hftoken